conf.yaml

```diff
@@ -1417,10 +1417,22 @@ contents:
                 repo:   enterprise-search-pubs
                 path:   workplace-search-docs
           -
-
     -
-        title:      SIEM: Security Operations with the Elastic Stack
+        title:      Elastic Security
         sections:
+          -
+            title:      Elastic Endpoint Security
+            prefix:     en/endpoint/guide
+            current:    7.6
+            branches:   [ master, 7.x, 7.6 ]
+            index:      docs/en/endpoint/index.asciidoc
```

Comment:
Probably want to define live branches here. If it's just `master` and `7.6`, you can use:
`live:       *stacklive`

Reply:
What does a "live" branch entail?

Reply:
Live means it shows up in the dropdown:
<img width="211" alt="Screen Shot 2020-02-20 at 4 34 45 PM" src="https://user-images.githubusercontent.com/5618806/74993130-eeb72500-53fe-11ea-8a52-a8dfe199fa01.png">
6.8 won't show as you don't have that branch.

Reply:
I'll skip that for now since the page we're adding is temporary.